test/integration/auth/node_test.go

```diff
@@ -284,8 +317,19 @@ func TestNodeAuthorizer(t *testing.T) {
 	expectForbidden(t, createNode2NormalPod(node2Client))
 	expectAllowed(t, updateNode2NormalPodStatus(node2Client))
 	expectAllowed(t, deleteNode2NormalPod(node2Client))
+	expectNotFound(t, createNode2NormalPodEviction(node2Client))
 	expectAllowed(t, createNode2MirrorPod(node2Client))
 	expectAllowed(t, deleteNode2MirrorPod(node2Client))
+	expectNotFound(t, createNode2MirrorPodEviction(node2Client))
+
+	expectAllowed(t, createNode2NormalPod(superuserClient))
+	expectAllowed(t, createNode2MirrorPod(node2Client))
+
+	// mirror/normal pod eviction and self node lifecycle is allowed
```

Comment:
if you do the create/delete,create/evict tests earlier, this set becomes unnecessary